components/slider/index.css

```diff
@@ -742,6 +746,36 @@ governing permissions and limitations under the License.
 
   }
 
+  /* Slider track background color doesn't show in high contrast mode, but border does */
+  .spectrum-Slider:not(.is-disabled) {
+    .spectrum-Slider-track {
+      &::before {
+        border-block-start: 2px solid transparent;
+      }
+    }
+
+    /* We only need these states in high contrast mode */
+    .spectrum-Slider-controls:hover,
+    .spectrum-Slider-controls:active,
+    .spectrum-Slider-controls:focus-within {
+      .spectrum-Slider-track::before {
+        background: var(--highcontrast-slider-track-color-hover, var(--mod-slider-track-color, var(--spectrum-slider-track-color)));
+        border-block-start: none;
+      }
```

Comment:
Hahah yes this is much better. Thank you!

The handle border-color is a bit different. We only have a hover effect set up on the handle itself, not the track. Which is fine without high contrast turned on because the track doesn't need a hover effect until we have WHCM.
So if I only update the `--highcontrast-slider-handle-border-color-hover` custom property, it does work, but we don't get the hover effect on the handle unless we hover it specifically. So the track turns the `Highlight` color but the handle is still white unless you're hovering the handle. I think this new solution matches the spec better, but I'm open to thoughts!